### ISSUE TITLE
Backviewsize About

### ISSUE THREAD

**Issue by AydroidDev** [pedromassango/IBackdrop#7]
Hi, I’m really grateful for this valuable library. But I have a problem. Toolbardan tab layout style for a menu I’m using a pole. I want to download the height of this layout up to the end of the table. With BackViewSize I can set the height I want with this correct. But when I try it on devices with different screen sizes, the front layout is either above or below the tabhost so the user cannot access the menu easily. In front of this for different screen sizes to create folders on different dp sdl and I tried to use that way again did not work. For example, a device with a size of 2.3 inch screen with a 3.7-inch screen size of the device I’m adjusting the same dp folder between them is even the difference. To sum up, I would like you to use the BackViewSize or use another method to set the same height when you set the height of all devices. Yes, I just want this, regardless of the screen size, for example, the front of the layout table at the end of all I wanted to stop at the same place. If you can handle this as soon as possible, I’ll update my application for my application. Thanks.

**Comment by pedromassango** [comment]
@AydroidDev can you please provide a screenshot about the issue? I don't have large devices to test the library, so please post some screenshot so that I can find a way to solve the issue. Thank you

**Comment by AydroidDev** [comment]
![img_20181215_013948_806](https://user-images.githubusercontent.com/25365989/50033637-61ae5a00-000a-11e9-8631-be39f3f60116.jpg)  or 
![screenshot_20181213-005134](https://user-images.githubusercontent.com/25365989/50033854-527bdc00-000b-11e9-9118-7a0bd5ceb200.png)

**Comment by AydroidDev** [comment]
Hi, for example, the first screenshot belongs to my 5-inch phone, while the second screen belongs to my 5.5-inch phone. Backdropsize is 540dp, so it looks like this on different devices with different screen sizes, and the Tab I'm using on one of them turns off the menu. To solve this, I used different dp s folders on the xml side, but it didn't work. Because two screen size w360dp is detecting the folder. I want to solve this problem.

**Comment by AydroidDev** [comment]
For example, I want the height of the entire screen to be up to the end of the tab bar menu, just like the second screen picture in height.

**Comment by AydroidDev** [comment]
Hey, Pedro, will I wait for this to be solved? All I want is to download the front layout to the end of the Tabbar, which appears in the pictures above. Even if I set the same height value, I would like to download all screens to the end of the tabbar so the space above disturbs me even though it is very noticeable even between the 5 inch and 5.2 inch phones.

**Comment by pedromassango** [comment]
Have you tried to set your Tabbar inside the Toolbar?

**Comment by halit4603** [comment]
Yeah, I tried. In fact, the TabBar I'm using is also a library, but I don't think the problem is about it. For example, in the description of your library, the layout will go down to half of the screen. Even if I do not interfere, I can download your library and try it in 3.2-inch and 5.5-inch emulators. I want it to go down to the end of the TabBar, rather than to go down to the middle of the screen. Think of something like xml as well as layout_behavior or layout_above functions. If this is not the case, it is enough for me to get a way that it is possible to stop the layout in front of the Tab after all devices. Thanks.

**Comment by pedromassango** [comment]
@halit4603 please try it and say something. I gonna try to calculate the Toolbar size then place he front layer below of it

**Comment by AydroidDev** [comment]
You're going to send me an update, I'm gonna try it, right?

**Comment by AydroidDev** [comment]
You're gonna send me an update.

**Comment by AydroidDev** [comment]
You will send an update, I will try it. You don't get it wrong, do you?

**Comment by pedromassango** [comment]
Yes I understood it wrong. Thanks, after I send an update I will comment here
